core/lib/utils.js

```diff
@@ -1,6 +1,22 @@
 'use strict';
 
+var _ = require('lodash');
```

Comment:
Please use [lodash.noConflict](https://lodash.com/docs#noConflict), just in case.